components/script/dom/webidls/DOMException.webidl

```diff
@@ -9,7 +9,7 @@
 
 [
   ExceptionClass,
-  Exposed=(Window,Worker),
+  Exposed=(Window,Worker,DissimilarOriginWindow),
```

Comment:
Try adding `Worklet` to this list as well. I suspect that is causing parse-input-arguments-005.https.html to fail.